src/main/java/org/apache/maven/report/projectinfo/TeamReport.java

```diff
@@ -52,19 +53,50 @@ public class TeamReport extends AbstractProjectInfoReport {
     /**
      * Shows avatar images for team members that have a) properties/picUrl set b) An avatar at gravatar.com for their
      * email address
-     * <p/>
-     * Future versions of this plugin may implement different strategies for resolving avatar images, possibly
-     * using different providers.
-     *<p>
-     *<strong>Note</strong>: This property will be renamed to {@code tteam.showAvatarImages} in 3.0.
+     *
      * @since 2.6
      */
     @Parameter(property = "teamlist.showAvatarImages", defaultValue = "true")
     private boolean showAvatarImages;
 
+    /**
+     * Indicate if external url should be used for avatar images.
```

Comment:
URL.
Is there an internal URL? If not, delete "external" here or rewrite

Reply:
I've rewritten a little here ...